buildprocess/configureWebpack.js

```diff
@@ -430,6 +434,11 @@ function configureWebpack(
     require.resolve("react-dom")
   );
 
+  config.resolve.alias["protomaps-leaflet"] = path.dirname(
+    require.resolve("protomaps-leaflet")
+  );
+  config.resolve.alias["pmtiles"] = path.dirname(require.resolve("pmtiles"));
+
```

Comment:
I'm not entirely sure why these need to be here - but the build was failing without them